src/coreclr/jit/importer.cpp

```diff
@@ -8508,10 +8494,8 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
 
         goto DONE;
     }
-    else if ((opcode == CEE_CALLI) && (((sig->callConv & CORINFO_CALLCONV_MASK) == CORINFO_CALLCONV_STDCALL) ||
-                                       ((sig->callConv & CORINFO_CALLCONV_MASK) == CORINFO_CALLCONV_C) ||
-                                       ((sig->callConv & CORINFO_CALLCONV_MASK) == CORINFO_CALLCONV_THISCALL) ||
-                                       ((sig->callConv & CORINFO_CALLCONV_MASK) == CORINFO_CALLCONV_FASTCALL)))
+    else if ((opcode == CEE_CALLI) && ((sig->callConv & CORINFO_CALLCONV_MASK) != CORINFO_CALLCONV_DEFAULT) &&
```

Comment:
Is it feasible to comment out or delete `CORINFO_CALLCONV_C`, and replace it with a comment that says to use `CorInfoCallConvExtension` and `getUnmanagedCallConv` ?

Reply:
`CORINFO_CALLCONV_C` and friends at this point are only used in the interpreter to detect NYI cases. I'll move them to the interpreter for now. I can swap them to the metadata constants if you'd prefer.

Reply:
> I can swap them to the metadata constants if you'd prefer.

Yes, that would be nice.